python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -870,6 +870,48 @@ def makeCandidateList(self, templateExposure, scienceExposure, kernelSize, candi
 
         return candidateList
 
+    def makeKernelBasisList(self, targetFwhmPix=None, referenceFwhmPix=None,
+                            basisDegGauss=None, basisSigmaGauss=None, metadata=None):
+        """Wrapper for `lsst.ip.diffim.makeKernelBasisList` to set log messages consistently.
+
+        Parameters
+        ----------
+        targetFwhmPix : `float`, optional
+            Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`.
+            Not used for delta function basis sets.
+        referenceFwhmPix : `float`, optional
+            Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`.
+            Not used for delta function basis sets.
+        basisDegGauss : `list` of `int`, optional
+            Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`.
+            Not used for delta function basis sets.
+        basisSigmaGauss : `list` of `int`, optional
+            Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`.
+            Not used for delta function basis sets.
+        metadata : `lsst.daf.base.PropertySet`, optional
+            Passed on to `lsst.ip.diffim.generateAlardLuptonBasisList`.
+            Not used for delta function basis sets.
+
+        Returns
+        -------
+        basisList: `list` of `lsst.afw.math.kernel.FixedKernel`
+            List of basis kernels.
+        """
+        basisList = makeKernelBasisList(self.kConfig,
+                                        targetFwhmPix=targetFwhmPix,
+                                        referenceFwhmPix=referenceFwhmPix,
+                                        basisDegGauss=basisDegGauss,
+                                        basisSigmaGauss=basisSigmaGauss,
+                                        metadata=metadata)
+        if targetFwhmPix == referenceFwhmPix:
+            self.log.info("Target and reference psf fwhms are equal, falling back to config values")
+        elif referenceFwhmPix > targetFwhmPix:
+            self.log.info("Reference psf fwhm is the greater, normal convolution mode")
+        else:
+            self.log.info("Target psf fwhm is the greater, deconvolution mode")
```

Comment:
So, if one was to run diffim in debug mode, does this mean you'll get double entries (one at `info` and one at `debug`)?

Reply:
It does, but I think we can deal with that since the logs can be long in debug mode.